generation/spade_gan/spade_gan.ipynb

```diff
@@ -0,0 +1,2665 @@
+{
```

Comment:
<p>We can remove the citations here, they have already been included in the second cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>